apps/ledger-live-desktop/tests/component/layout.component.ts

```diff
@@ -55,11 +55,16 @@ export class Layout extends Component {
     await this.drawerMarketButton.click();
   }
 
-  @step("Check if there is a input error visible")
+  @step("Check if there is no input error visible")
   async checkInputErrorNotVisible() {
     await this.inputError.waitFor({ state: "hidden" });
   }
 
+  @step("Check if there is a input error visible")
+  async checkInputErrorVisible() {
+    await this.inputError.waitFor({ state: "visible" });
+  }
+
```

Comment:
```suggestion
@step("Check input error state visibibility: $0")
  async checkInputErrorVisibibility(expectedState: "visible" | "hidden") {
    await this.inputError.waitFor({ state: expectedState });
  }
```

this could potentially replace both methods above